### PR TITLE
Updating citrix-istio-adaptor helm charts

### PR DESCRIPTION
**Pull request by subashd**
Please add this, as we want to list our helm charts in helm hub.

### CODE REVIEW

**Comment by subashd** [comment]
@mattfarina I have signed the commits... could you please review the PR?

**Comment by mattfarina** [comment]
@subashd I think there is a problem with your git configuration. The error says...

> Commit sha: f5753b1, Author: Subash Dangol, Committer: Subash Dangol; Expected "Subash Dangol you@example.com", but got "Subash Dangol subash.dangol@citrix.com".

The email addresses should match. See https://help.github.com/en/articles/setting-your-commit-email-address#setting-your-commit-email-address-in-git for an example of how to set the email. Please use your email address here.

**Comment by subashd** [comment]
@mattfarina could you please review the PR?

File: config/repo-values.yaml [hunk 1]
```diff
@@ -72,8 +72,10 @@ sync:
       url: https://hazelcast.github.io/charts/
     - name: nginx
       url: https://helm.nginx.com/stable
-    - name: cic
+    - name: citrix
       url: https://citrix.github.io/citrix-k8s-ingress-controller/
+    - name: citrix
```
Comment (mattfarina) [inline config/repo-values.yaml]: Names need to be unique for each repository. Can you please use two different unique names.

**Comment by subashd** [comment]
@mattfarina  i have fixed the issue.

**Pull request merged**